lib/src/child-writer/index.js

```diff
@@ -56,20 +57,50 @@ const EXIT_CODES = require('../exit-codes');
 exports.write = (image, drive, options) => {
   const emitter = new EventEmitter();
 
-  utils.getTemporaryLogFilePath().then((logFile) => {
-    const argv = utils.getCLIWriterArguments({
-      entryPoint: rendererUtils.getApplicationEntryPoint(),
-      logFile: logFile,
-      image: image,
-      device: drive.device,
-      validateWriteOnSuccess: options.validateWriteOnSuccess,
-      unmountOnSuccess: options.unmountOnSuccess
-    });
+  const argv = utils.getCLIWriterArguments({
+    entryPoint: rendererUtils.getApplicationEntryPoint(),
+    image: image,
+    device: drive.device,
+    validateWriteOnSuccess: options.validateWriteOnSuccess,
+    unmountOnSuccess: options.unmountOnSuccess
+  });
+
+  // There might be multiple Etcher instances running at
+  // the same time, therefore we must ensure each IPC
+  // server/client has a different name.
+  process.env.IPC_SERVER_ID = `etcher-server-${process.pid}`;
+  process.env.IPC_CLIENT_ID = `etcher-client-${process.pid}`;
+
+  ipc.config.id = process.env.IPC_SERVER_ID;
+  ipc.config.silent = true;
+  ipc.serve();
+
+  ipc.server.on('error', (error) => {
+    emitter.emit('error', error);
+  });
+
+  ipc.server.on('message', (data) => {
+    const message = JSON.parse(data);
```

Comment:
Could this be surrounded in a `try...catch` which would also emit an error, in the same way that not having a `command` or `data` property would?